src/stylesheets/settings/_settings-components.scss

```diff
@@ -80,16 +80,14 @@ $theme-footer-max-width: "desktop" !default;
 // Form and input
 $theme-checkbox-border-radius: "sm" !default;
 $theme-form-font-family: "ui" !default;
+$theme-input-background-color: default !default;
 $theme-input-line-height: 3 !default;
 $theme-input-max-width: "mobile-lg" !default;
 $theme-input-select-border-width: 2px !default;
 $theme-input-select-size: 2.5 !default;
 $theme-input-state-border-width: 0.5 !default;
-$theme-input-tile-background-color-selected: "primary-lighter" !default;
 $theme-input-tile-border-radius: "md" !default;
 $theme-input-tile-border-width: 2px !default;
-$theme-input-tile-border-color: "base-lighter" !default;
-$theme-input-tile-border-color-selected: "primary-light" !default;
```

Comment:
The system now uses the standard or reverse link color as the selected color instead of a special new color.